bindings/prql-php/phpstan.neon

```diff
@@ -0,0 +1,13 @@
+parameters:
+    level: 9
+    paths:
+        - src
+        - tests
+    ignoreErrors:
```

Comment:
```suggestion
    ignoreErrors:
    Since [PHPStan](https://phpstan.org/) doesn't know about the existence of the functions in libprql, it complains since they get dynamically invoked on the FFI object instance without being statically defined, so they're unknown.
```